modules/apps/dynamic-data-mapping/dynamic-data-mapping-lang/src/main/resources/content/Language.properties

```diff
@@ -410,7 +410,7 @@ the-setting-x-set-for-field-x-is-invalid=Setting ({0}) of field {1} is invalid.
 the-settings-x-set-for-field-x-are-invalid=Settings ({0}) of field {1} are invalid.
 the-small-image-file-could-not-be-saved=The small image file could not be saved.
 the-specified-redirect-url-is-not-allowed=The specified redirect URL is not allowed.
-the-specified-redirect-url-is-not-allowed-due-to-installation-settings.-add-x-into-the-property-x-to-fix-it=The specified redirect URL is not allowed due to installation settings. Add "{0}" into the property "{1}" to fix it.
+the-specified-redirect-url-is-not-allowed-due-to-installation-settings.-add-x-into-the-property-x-to-fix-it=The External Redirect URL {0} is not allowed. Set it in the portal property {1} to allow it.
```

Comment:
@mlan0, can you please update the language key according to the new language value?

I mean rename to `the-external-redirect-url-x-is-not...`